ci/setup-build-test-gdk.sh

```diff
@@ -45,26 +44,25 @@ pushd "$(dirname "$0")"
         "${BUILD_STATE}" \
         "${TEST_UPROJECT_NAME}${BUILD_TARGET}"
 
-    # TODO UNR-3164 - re-enable tests after we made sure they work for Mac
-    # echo "--- run-fast-tests"
-    # "${GDK_HOME}/ci/run-tests.sh" \
-    #     "${UNREAL_PATH}" \
-    #     "${TEST_PROJECT_NAME}" \
-    #     "${UPROJECT_PATH}" \
-    #     "FastTestResults" \
-    #     "NetworkingMap" \
-    #     "SpatialGDK+/Game/SpatialNetworkingMap" \
-    #     "True"
+    echo "--- run-fast-tests"
+    "${GDK_HOME}/ci/run-tests.sh" \
+        "${UNREAL_PATH}" \
+        "${TEST_PROJECT_NAME}" \
+        "${UPROJECT_PATH}" \
+        "TestResults" \
+        "${TEST_REPO_MAP}" \
+        ".SpatialGDK" \
+        "True"
 
-    # if [[ -n "${SLOW_NETWORKING_TESTS}" ]]; then
-    #     echo "--- run-slow-networking-tests"
-    #     "${GDK_HOME}/ci/run-tests.sh" \
-    #         "${UNREAL_PATH}" \
-    #         "${TEST_PROJECT_NAME}" \
-    #         "${UPROJECT_PATH}" \
-    #         "VanillaTestResults" \
-    #         "NetworkingMap" \
-    #         "+/Game/NetworkingMap" \
-    #         ""
-    # fi
+    if [[ -n "${SLOW_NETWORKING_TESTS}" ]]; then
+        echo "--- run-slow-networking-tests"
+        "${GDK_HOME}/ci/run-tests.sh" \
+            "${UNREAL_PATH}" \
+            "${TEST_PROJECT_NAME}" \
+            "${UPROJECT_PATH}" \
+            "SlowTestResults" \
+            "${TEST_REPO_MAP}" \
+            "+SpatialGDKSlow." \
```

Comment:
```suggestion
            "SpatialGDKSlow." \
```
This should be enough, if launching two separate processes.
An alternative would be to call SpatialGDK.+SpatialGDKSlow. in one process if SLOW_NET_TEST